### ISSUE TITLE
Installation fails because of timeout on Raspberry Pi

### ISSUE THREAD

**Issue by jankais3r** [performancecopilot/pcp#705]
The package can't be installed on the first gen Raspberry Pi (Raspbian 10 Buster), because the config phase always times out. Would it be possible to increase the timeout?

```
$ sudo apt install pcp
[…]
Setting up pcp (4.3.2+really4.3.1-0.1+b1) ...
Job for pmlogger.service failed because a timeout was exceeded.
See "systemctl status pmlogger.service" and "journalctl -xe" for details.
invoke-rc.d: initscript pmlogger, action "start" failed.
● pmlogger.service - Performance Metrics Archive Logger
   Loaded: loaded (/lib/systemd/system/pmlogger.service; enabled; vendor preset: enabled)
   Active: activating (auto-restart) (Result: timeout) since Wed 2019-07-24 14:07:37 CEST; 196ms ago
     Docs: man:pmlogger(1)

Jul 24 14:07:38 torpi systemd[1]: pmlogger.service: Service RestartSec=100ms expired, scheduling restart.
Jul 24 14:07:38 torpi systemd[1]: pmlogger.service: Scheduled restart job, restart counter is at 3.
Jul 24 14:07:38 torpi systemd[1]: Stopped Performance Metrics Archive Logger.
dpkg: error processing package pcp (--configure):
 installed pcp package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 pcp
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

**Comment by jankais3r** [comment]
Workaround that worked for me was adding ’TimeoutStartSec=infinity` to `/lib/systemd/system/pmlogger.service’ and then manually running `sudo dpkg --configure -a`

**Comment by goodwinos** [comment]
Hi Janaiks3r,

thanks for the report

commit 0df9f82aa74f20418acd14 added 120 timeouts to all PCP system service config (for both start and stop). This should help with this issue if you could re-test.  Thanks

**Comment by goodwinos** [comment]
Whether this was a genuine timeout (Rasp PIs are slow systems), or whether it was a known bug in the PCP rc scripts ... either way this should be fixed now (especially in the soon to be released 5.1.1 release).
